### DISCUSSION TITLE
Issue while creating ONLINE indexes for localhost.

### DISCUSSION THREAD

**Discussion by anuksap** [grate-devs/grate#576] [Q&A]
Hello Team,
I was trying to migrate from rh to grate.
So I executed grate command locally to see changes on localhost . But it got failed for online indexes.
When I did ONLINE = OFF it works. But rh works totally fine for Online = ON.
Could you please assist if anything I am missing.

Below are the snippets from logs
Online index operations can only be performed in Enterprise edition of SQL Server or Azure SQL Edge.

**Comment by PagesplitPeter** [comment]
@anuksap that is by design of SQL Server and not something Grate can change.

Doing a index create or rebuild with Index ONLINE = ON is an enterprise feature

Microsoft has publish the difference between the editions on [Editions and supported features of SQL Server 2022](https://learn.microsoft.com/en-us/sql/sql-server/editions-and-components-of-sql-server-2022?view=sql-server-ver16&preserve-view=true)
